images/scripts/opensuse.setup

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
This lacks a counterpart in the form of `opensuse.bootstrap` Or is this supposed to be for `opensuse-tumbleweed`?

Reply:
I seem to have missed a symlink it has been fixed